src/server/rfm/rfm_funcs/create_bins.py

```diff
@@ -0,0 +1,54 @@
+def create_bins(data, query_date):
+    '''This script will take table data and bin edges for RFM scores for all PAWS donations
+
+    query_date = date data was queried
+    '''
+
+    import pandas as pd
+    import numpy as np
+    import jenkspy
+    from datetime import datetime, date
+    import os
+
+
+
+    ####
+    # read in csv
+    donations_df = pd.read_csv('csv')

```

Comment:
Aren't we getting data from tables (via `pull_donations_for_rfm()`), not from files?